source/elements/oneDAL/source/data_management/index.rst

```diff
@@ -126,172 +130,137 @@ used at the data preparation and data processing stages for the following:
+
+  When a table represents a dataset, its rows represent observations in the
+  dataset, and its columns represent features, according to the
+  :txtref:`dataset` definition.
 
 This concept has different logical organization and physical :capterm:`format of
 the data <data format>`:
 
-- Logically, a table is a :txtref:`dataset` with :math:`n` rows and
-  :math:`p` columns. Each row represents an :capterm:`observation` and each
-  column is a :capterm:`feature` of a dataset. Physical amount of bytes needed
-  to store the data differ from the number of elements :math:`n \times p` within
-  a table.
+- Logically, a table contains :math:`n` rows and :math:`p` columns.
+  Every column may have its own type of data values and a set of allowed
+  operations.
 
 - Physically, a table can be organized in different ways: as a
   :capterm:`homogeneous <homogeneous data>`, :capterm:`contiguous <contiguous
   data>` array of bytes, as a :capterm:`heterogeneous <heterogeneous data>` list
   of arrays of different :capterm:`data types <data type>`, in a
-  compressed-sparse-row format.
+  compressed-sparse-row format. The number of bytes needed to store the data
+  differs from the number of elements :math:`n \times p` within a table.
 
 For details, see :txtref:`tables` section.
 
-.. _metadata:
-
-Metadata
---------
-
-Metadata concept is associated with a :txtref:`dataset` and holds information
-about its structure and type. This information shall be enough to determine the
-particular type of a dataset, and it helps to understand how to interact with a
-dataset in |dal_short_name| (for example, how to use it at a particular stage of
-:txtref:`data analytics pipeline <data_analytics_pipeline>` or how to access its
-data).
-
-For each dataset, its metadata shall contain:
+.. _table_metadata:
 
-- The number of rows :math:`n` and columns :math:`p` in a dataset.
+Table metadata
+--------------
 
-- The type of each :capterm:`feature` (e.g. :capterm:`nominal <nominal
-  feature>`, :capterm:`interval <interval feature>`).
+Table metadata concept provides an additional information about data in the
+table:
 
-- The :capterm:`data type` of each feature (e.g. :code:`float` or
-  :code:`double`).
-
-.. note::
-  Metadata can contain both compile-time and run-time information. For example,
-  basic compile-time metadata is the type of a dataset - whether it is a
-  particular :txtref:`data-source` or a :txtref:`table`. Run-time information
-  can contain the :capterm:`feature` types and :capterm:`data types <data type>`
-  of a dataset.
+1. The `data types <Data type>` of the columns.
 
-.. _table-builder:
+2. The logical types of data in the columns:
+   :capterm:`nominal <Nominal feature>`, :capterm:`ordinal <Ordinal feature>`,
+   :capterm:`interval <Interval feature>`, or :capterm:`ratio <Ratio feature>`.
+   Information about logical types is mostly relevant in cases when table
+   columns represent dataset features, but it is also applicable in other cases
```

Comment:
I'd just drop the sentence 'information about ...' as it follows from this sentence that logical types can be used either case